examples/Tutorial/Tutorial.cpp

```diff
@@ -147,6 +147,12 @@ class GameWindow : public Gosu::Window
     {
         set_caption("Gosu Tutorial Game");
 
+        {
+            Gosu::Bitmap icon;
```

Comment:
I'm not sure if this is leftover test code, but I think it seems reasonable enough to have this in the Tutorial. But it should not have the extra scope nesting :)